lib/lightning_web/live/workflow_live/workflow_params.ex

```diff
@@ -13,7 +13,13 @@ defmodule LightningWeb.WorkflowNewLive.WorkflowParams do
     Map.merge(current_params, form_params, fn _key, current, next ->
       case {current, next} do
         {current, next} when is_list(current) and is_map(next) ->
-          merge_list_params(current, next)
+          if Map.has_key?(form_params, "edges") &&
```

Comment:
i think this is the code that's setting downstream edges. please ensure that if someone disbales an edge, only that edge gets disabled.